[2024-05-12T20:58:40.841+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-12T20:58:40.895+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Data_Extraction_Transformation_Loading.transform_data manual__2024-05-12T20:40:30.453200+00:00 [queued]>
[2024-05-12T20:58:40.922+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Data_Extraction_Transformation_Loading.transform_data manual__2024-05-12T20:40:30.453200+00:00 [queued]>
[2024-05-12T20:58:40.923+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-12T20:58:40.987+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-05-12 20:40:30.453200+00:00
[2024-05-12T20:58:41.004+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Data_Extraction_Transformation_Loading', 'transform_data', 'manual__2024-05-12T20:40:30.453200+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/i202478_dag.py', '--cfg-path', '/tmp/tmpe38l8zk0']
[2024-05-12T20:58:41.009+0000] {standard_task_runner.py:91} INFO - Job 167: Subtask transform_data
[2024-05-12T20:58:41.015+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=227) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-12T20:58:41.017+0000] {standard_task_runner.py:63} INFO - Started process 236 to run task
[2024-05-12T20:58:41.161+0000] {task_command.py:426} INFO - Running <TaskInstance: Data_Extraction_Transformation_Loading.transform_data manual__2024-05-12T20:40:30.453200+00:00 [running]> on host 3a6423a4af12
[2024-05-12T20:58:41.392+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Data_Extraction_Transformation_Loading' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-12T20:40:30.453200+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-12T20:40:30.453200+00:00'
[2024-05-12T20:58:41.394+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-12T20:58:41.521+0000] {logging_mixin.py:188} WARNING - /opt/airflow/dags/i202478_dag.py:207 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-12T20:58:41.548+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-12T20:58:41.550+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-12T20:58:41.586+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Data_Extraction_Transformation_Loading, task_id=transform_data, run_id=manual__2024-05-12T20:40:30.453200+00:00, execution_date=20240512T204030, start_date=20240512T205840, end_date=20240512T205841
[2024-05-12T20:58:41.677+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-12T20:58:41.747+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-12T20:58:41.761+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
